ebean-core/src/test/java/org/tests/query/TestQueryExists.java

```diff
@@ -89,26 +89,26 @@ public void testExists_onOneToMany() {
   public void testExists() {
     ResetBasicData.reset();
 
-    Query<Order> subQuery = Ebean.find(Order.class).alias("sq").select("id").where().raw("sq.kcustomer_id = qt.id").query();
+    Query<Order> subQuery = DB.find(Order.class).alias("sq").select("id").where().raw("sq.kcustomer_id = qt.id").query();
 
-    Query<Customer> query = Ebean.find(Customer.class).alias("qt").where().exists(subQuery).query();
+    Query<Customer> query = DB.find(Customer.class).alias("qt").where().exists(subQuery).query();
```

Comment:
removed the `.select("id")` from all exists subqueries and refactored `Ebean` to `DB`